bpython/curtsiesfrontend/manual_readline.py

```diff
@@ -231,6 +231,25 @@ def delete(cursor_offset, line):
     return (cursor_offset, line[:cursor_offset] + line[cursor_offset + 1 :])
 
 
+def cursor_on_closing_char_pair(cursor_offset, line):
+    """Checks if cursor sits on closing character of a pair
+    and whether its pair character is directly behind it
+    """
+    closing_char_map = {"(": ")", "{": "}", "[": "]", "'": "'", '"': '"'}
```

Comment:
Since the same map is used as `BaseRepl.PAIR_CHAR_MAP`, this could be defined once and the same map used everywhere.

Reply:
i think i left this there to avoid circular import, since manual_readline file is imported in repl.py

Reply:
You could move `cursor_on_closing_char_pair` and `PAIR_CHAR_MAP` to `bpython/line.py` where all the other functions of this type are.

Reply:
so it should propably go outside of this both files

Reply:
okay. i also moved the MAP to __init__.py